packages/embeds/embed-core/playwright/tests/inline.e2e.ts

```diff
@@ -6,29 +6,23 @@ import {
   assertNoRequestIsBlocked,
   bookFirstEvent,
   deleteAllBookingsByEmail,
-  getEmbedIframe,
+  ensureEmbedIframe,
 } from "../lib/testUtils";
 
 test.describe("Inline Iframe", () => {
-  test("Inline Iframe - Configured with Dark Theme. Do booking and verify that COEP/CORP headers are correctly set", async ({
+  test("Configured with Dark Theme. Do booking and verify that COEP/CORP headers are correctly set", async ({
     page,
-    embeds: { addEmbedListeners, getActionFiredDetails },
+    embeds,
   }) => {
     await deleteAllBookingsByEmail("embed-user@example.com");
-    await addEmbedListeners("");
-    await page.goto("/?only=ns:default");
+    await embeds.gotoPlayground({ calNamespace: "", url: "/?only=ns:default" });
     const calNamespace = "";
-    const embedIframe = await getEmbedIframe({ calNamespace, page, pathname: "/pro" });
-    expect(embedIframe).toBeEmbedCalLink(calNamespace, getActionFiredDetails, {
-      pathname: "/pro",
```

Comment:
No need to check for pathname again. We are getting the embed iframe by pathname only.